docs/events-actions.md

```diff
@@ -0,0 +1,368 @@
+# Events and Actions
```

Comment:
Before this you need to link it with the order of the other docs:
```
---
next: docs/http.md
---
```

Also I propose that this doc goes after the github-api page, which means that the top of that files needs to become:
```
---
next: docs/events-actions.md
---
```

However, I'm open to other thoughts on where exactly to place it in the order of the docs. Thoughts @gr2m?

Reply:
yes, good thinking, I agree

Reply:
Added events-action.md between github-api.md and http.md by updating next links. 😄